### PR TITLE
Step 1 of getting paasta_native services into smartstack: implement get_paasta_native_services_running_here_for_nerve

### PR DESCRIPTION
**Pull request by EvanKrall**

### CODE REVIEW

File: paasta_tools/native_mesos_scheduler.py [hunk 1]
```diff
@@ -527,6 +529,86 @@ def get_mesos_network_mode(self):
...
+    in the 'test.main' namespace as well as the 'other_svc.main' namespace.
+
+    If one is not defined in the config file, returns a list containing
+    name.instance instead.
+    """
+    if not cluster:
+        cluster = load_system_paasta_config().get_cluster()
+
+    job_config = load_paasta_native_job_config(
+        service, instance, cluster, load_deployments=False, soa_dir=soa_dir
```
Comment (solarkennedy) [inline paasta_tools/native_mesos_scheduler.py]: please use kwargs here to avoid arg ordering bugs

File: paasta_tools/marathon_tools.py [hunk 2]
```diff
@@ -750,27 +749,6 @@ def get_classic_services_running_here_for_nerve(soa_dir):
     return classic_services
 
 
-def get_services_running_here_for_nerve(cluster=None, soa_dir=DEFAULT_SOA_DIR):
```
Comment (solarkennedy) [inline paasta_tools/marathon_tools.py]: We don't need this function anymore?
Or are we doing this inside of synapse tools? (the concatenation of all the frameworks and junk)

Comment (EvanKrall) [inline paasta_tools/marathon_tools.py]: https://github.com/Yelp/nerve-tools/pull/23#discussion_r90557832

**Comment by solarkennedy** [comment]
@EvanKrall good to go

**Pull request merged**